thrall/app/controllers/ThrallController.scala

```diff
@@ -90,10 +99,23 @@ class ThrallController(
               apiBaseUrl = services.apiBaseUri,
               uiBaseUrl = services.kahunaBaseUri,
               filter,
-              page
+              page,
+              shouldAllowReattempts = true
             ))
           )
-        case _ => Future.successful(Ok("No current migration"))
+        case _ => for {
+          currentIndex <- es.getIndexForAlias(es.imagesCurrentAlias)
+          currentIndexName <- currentIndex.map(_.name).map(Future.successful).getOrElse(Future.failed(new Exception(s"No index found for '${es.imagesCurrentAlias}' alias")))
+          failures <- es.getMigrationFailures(es.imagesHistoricalAlias, currentIndexName, from, pageSize, filter)
```

Comment:
This throws (I see an error page) if the historical alias doesn't exist

Reply:
i think that's fine, no?